assets/css/style.css

```diff
@@ -85,60 +87,114 @@ p {
   padding-left: 45px;
 }
 
+/*current navigation - in progress */
 
-
-  /*current navigation - in progress */
-.top-nav {
-  color: #000000;
-  font-size: 20px;
-  text-decoration: none;
+.nav {
+  display: block;
 }
 
-
-ul,
-li {
+.nav-li {
+  padding: 20px 0px;
   list-style: none;
-  display: inline;
-  overflow: hidden;
 }
-li {
-  padding: 50px;
+
+.menu {
+  display: flex;
+  position: absolute;
+  justify-content: space-between;
+  left: 100%;
+  right: 0;
+  top: 5.2rem;
+  flex-direction: column;
+  background-color: lavender;
+  border-radius: 20px;
+  text-align: center;
+  transition: 0.4s;
+  margin-left: 0px;
+  padding: 0px 150px;
+  opacity: 95%;
+}
+.hamburger-icon,
+.logo {
+  display: block;
+  cursor: pointer;
+  position: absolute;
 }
 
-a {
-  text-decoration: none;
+.hamburger-icon {
+  right: 25px;
+  top: 1rem;
 }
 
 .hamburger {
   display: block;
-  z-index: 1;
-  width: 38px;
-  height: 5px;
-  margin-bottom: 5px;
-  position: relative;
-  background: #371a7a;
-  border-radius: 4px;
+  width: 65px;
+  height: 10px;
+  margin: 0px 0px 12px 20px;
+  background-color: #fff;
+  border-radius: 13px;
+}
+
+.menu.active {
+  left: 0;
+}
+.top-nav {
+  color: #000000;
+  font-size: 18px;
+  text-decoration: none;
 }
-.img-home {
-  width: 200px; /*PLACEHOLDER*/
+a {
+  text-decoration: none;
 }
 
+/* ---------MEDIA CSS----------
 /* Medium devices (tablets, 768px and up) */
 @media (min-width: 768px) {
+  .hamburger-icon {
+    display: none;
+  }
+
   p {
     font-size: 20px;
   }
+
+  .nav {
+    display: inline;
+  }
+
   a:hover {
-    font-weight: 700;
+    font-weight: 600;
```

Comment:
I would suggest instead of a text color change and outline, we just bold the copy like we have for active page links. It brings more attention to the hover effect since its a higher contrast in color than the purple. I'm also fine if you want to keep it purple, but ditch the underline and add a higher weight. 700 is for bold.

Reply:
adjusted link font